src/e2e/puppeteer/__tests__/color.ts

```diff
@@ -163,3 +164,30 @@ it('Empty <span> element will be removed after setting color to default.', async
   const result = await getEditingText()
   expect(result).toBe('Golden Retriever')
 })
+
+it('Set the superScript color of the text', async () => {
```

Comment:
This test title doesn't makes sense. Maybe something like "Set the superscript color to the color of the whole thought".